### PR TITLE
Rspec view

### PR DESCRIPTION
**Pull request by phamducpho1**

### CODE REVIEW

**Comment by phamducpho1** [comment]
image

**Comment by phamducpho1** [comment]
ready

File: Gemfile [hunk 1]
```diff
@@ -25,6 +25,7 @@ gem "coffee-rails", "~> 4.2"
 gem "turbolinks", "~> 5"
 gem "jbuilder", "~> 2.5"
 gem "pry-byebug"
+gem "simplecov", :require => false, :group => :test
```
Comment (dothidiemthao) [inline Gemfile]: Ko dùng : => em

**Comment by dothidiemthao** [comment]
commented

**Comment by phamducpho1** [comment]
ready

**Pull request merged**